crates/core/src/client.rs

```diff
@@ -89,6 +89,18 @@ pub struct BeerusClient {
     /// Interval at which beerus will check if the L1 State Root
     /// has been updated.
     pub poll_secs: u64,
+    /// Timeout Seconds
+    ///
+    /// Timeout for fetching L1/L2 state root
+    pub timeout_secs: u64,
```

Comment:
Imho, for the current style, hard-coding values within a some resolving scope seem to be sufficient without complicating the configuration

Reply:
Fixed. They are being hard-coded now in the `wrapped_retry` function. I will resolve this conversation but feel free to reopen it ;)